File: practices/time_of_day.py
# LA time of day 
hour = int(input("What is the hour in military time? "))

if hour <= 11 and hour >= 0:
    print("Good morning!")
elif hour > 11 and hour <= 16:
    print("Good afternoon!")
elif hour > 16 and hour <= 24:
    print("Good evening!")
else:
    print(f"{hour} is not a military time. Maybe go look up a time chart.")

File: practices/time_of_day.c
//LA time of day in C
#include <stdio.h>
#include <string.h>

int main(){
    int hour;

    printf("What is the hour in military time?\n");
    scanf("%d", &hour);

    if (hour <= 11 && hour >= 0){
        printf("Good morning!\n");
    }else if (hour > 11 && hour <= 16){
        printf("Good afternoon!\n");
    }else if (hour > 16 && hour <= 24){
        printf("Good evening!\n");
    }else{
        printf("%d is not a military time. Maybe go look up a time chart.\n", hour);
    }
    return 0;
}